libs/solid/quiz/src/lib/components/drag-and-drop-choice-question/drag-and-drop-question.component.ts

```diff
@@ -1,18 +1,96 @@
-import { Component, Input } from '@angular/core';
-import { DragAndDropQuestion } from '../../state/quiz.model';
+import { Component, Input, OnInit } from '@angular/core';
+import {CdkDragDrop, moveItemInArray} from '@angular/cdk/drag-drop';
+import { DragAndDropQuestion, QuizAnswer } from '../../state/quiz.model';
 import { QuestionTypeComponent } from '../question/question.component';
 
+
+class Status {
+  constructor(public text: string, public status: string) {
+  }
+}
+
 @Component({
   selector: 'solid-quiz-drag-and-drop-question',
   templateUrl: './drag-and-drop-question.component.html',
   styleUrls: ['./drag-and-drop-question.component.scss'],
 })
-export class DragAndDropQuestionComponent implements QuestionTypeComponent {
+export class DragAndDropQuestionComponent implements QuestionTypeComponent, OnInit {
   @Input() public question!: DragAndDropQuestion;
```

Comment:
The `question` propertie has the model with the data for the component. Please use it instead of the static movie list.